### DISCUSSION TITLE
Custom Input Field using input text on outside table

### DISCUSSION THREAD

**Discussion by agustinus-robert** [rappasoft/laravel-livewire-tables#1350] [Q&A]
Hei, i want to make custom input field like regular datatables and custom view on my filter because it using class on bootstrap too, but using  livewire table.

(Sorry my bad englishh)

Can livewire table available using custom filter input... And one more thing, the filter is outside table too

Here i give screenshot, that i want too

![Screenshot 2023-09-09 050328](https://github.com/rappasoft/laravel-livewire-tables/assets/61762097/bce00e4b-232c-4ace-aca0-86efa8a5ba5a)

And this is my input text with custom css, i give you one example input field

![Screenshot 2023-09-09 050933](https://github.com/rappasoft/laravel-livewire-tables/assets/61762097/7e1f4ae1-0b5d-47f3-85c2-2294aa49accb)

**Comment by lrljoe** [comment]
Apologies for the delay in coming back to you, but I've been away.

Just so that I'm clear on what you are asking.

Do you want:
1) To be able to customise the class/look/feel of the existing filters?
2) Be able to render the filters in a non-standard location (e.g. above the toolbar)
3) Be able to use filters outside of the table component?

No 1 and No 2 are straight forward to achieve, No 3 is slightly less so at present, although is somewhat possible with the v3 branch.

If you are just looking to add a "search" for various columns, then this is supported by default, and you can see great examples in the documentation on [https://rappasoft.com/docs/laravel-livewire-tables/v2/introduction](https://rappasoft.com/docs/laravel-livewire-tables/v2/introduction)

**Reply by agustinus-robert** [reply]
yeah, i need number 3... mybe i should try v3

**Reply by agustinus-robert** [reply]
can you show me how to do it, on number 3 using livewire v3..

**Reply by lrljoe** [reply]
#3 is very much in beta, so not fully supported, but happy to give you pointers on the official Discord, just not posting them on here right now as it'll open up a world of support hurt for me
